data/scripts/actions/others/flower_pot.lua

```diff
@@ -54,14 +54,14 @@ local flowerPot = Action()
 function flowerPot.onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	for _, flower in pairs(flowers) do
 		if target.itemid == flower.itemid then
-			if (flower.watered == false and flower.advance == false) then
+			if (not flower.watered and not flower.advance) then
 				player:say(flower.msg, TALKTYPE_MONSTER_SAY)
-			elseif (flower.watered == true and flower.advance == false) then
+			elseif (flower.watered and not flower.advance) then
 				target:transform(flower.after)
 				player:say(flower.msg, TALKTYPE_MONSTER_SAY)
 				toPosition:sendMagicEffect(CONST_ME_LOSEENERGY)
 				target:decay()
-			elseif (flower.watered == true and flower.advance == true) then
+			elseif (flower.watered and flower.advance) then
```

Comment:
While at that, I believe none of these need the parenthesis

Reply:
Thanks!